src/appleseed/renderer/kernel/lighting/lighttypes.h

```diff
@@ -43,6 +43,8 @@
 // Forward declarations.
 namespace renderer  { class AssemblyInstance; }
 namespace renderer  { class BackwardLightSampler; }
+namespace renderer  { class ForwardLightSampler; }
+namespace renderer  { class LightSamplerBase; }

```

Comment:
Order alphabetically.

Reply:
Isn't it ?

Reply:
Nope :)

Reply:
![image](https://user-images.githubusercontent.com/4656466/57913408-d709a980-788c-11e9-8dd0-3b4cae86beb3.png)
Wait, what ? XD

Reply:
Ah! I wasn't seeing what was under your comment...